src/com/palmergames/bukkit/towny/TownyMessaging.java

```diff
@@ -28,6 +29,7 @@
 public class TownyMessaging {
 	private static final Logger LOGGER = LogManager.getLogger(Towny.class);
 	private static final Logger LOGGER_DEBUG = LogManager.getLogger("com.palmergames.bukkit.towny.debug");
+	private static final String PREFIX = TownySettings.getLangString("default_towny_prefix");

```

Comment:
This would be really convenient to have, however, it prevents changing the prefix on language file reload.